modules/serverless-https-redirect-global.adoc

```diff
@@ -1,13 +1,14 @@
 // Module included in the following assemblies:
 //
-// * /serverless/admin_guide/knative-serving-CR-config.adoc
+// * serverless/admin_guide/serverless-configuration.adoc
 
 :_content-type: REFERENCE
 [id="serverless-https-redirect-global_{context}"]
 = HTTPS redirection global settings
 
-You can enable HTTPS redirection for all services on the cluster by configuring the `httpProtocol` spec for the `KnativeServing` custom resource, as shown in the following example:
+HTTPS redirection provides encryption and redirection for incoming HTTP requests. You can enable HTTPS redirection for all services on the cluster by configuring the `httpProtocol` spec for the `KnativeServing` custom resource (CR).
```

Comment:
```suggestion
HTTPS redirection provides redirection for incoming HTTP requests. The redirected HTTP requests will be encrypted. You can enable HTTPS redirection for all services on the cluster by configuring the `httpProtocol` spec for the `KnativeServing` custom resource (CR).
```

The HTTPS redirection only provides redirection actually. But as a result, the redirected HTTP requests will be encrypted (because it starts using secure port). So, I guess that we should write it a little bit accurately.